jupysql_plugin/server_handlers/dashboard.py

```diff
@@ -8,7 +9,11 @@
 import requests
 import os
 
-BACKEND_ENDPOINT = "https://cloudapi.ploomber.io"
+PLOOMBER_CLOUD_HOST = os.environ.get(
+    "PLOOMBER_CLOUD_HOST", "https://cloudapi.ploomber.io"
+)
+logger = logging.getLogger(__name__)
+logger.info(f"Using this cloud host: {PLOOMBER_CLOUD_HOST}")
```

Comment:
I don't think this will ever be printed, since we're not configuring a logger in this project (and I don't think we should)

Reply:
I'm not sure what's the best idea here. maybe I'm wrong and this will actually be visible (?). this runs in the jupyter server so I don't know if jupyter configures a logger by default